addon/types/index.ts

```diff
@@ -3,10 +3,10 @@ export type WaiterName = string;
 export interface IWaiter {
```

Comment:
Since we don't have a class for this, maybe we can just call this one `Waiter`

Reply:
I'd prefer to keep our interface naming consistent, if you don't mind.